rails/app/javascript/components/Map.jsx

```diff
@@ -57,6 +58,28 @@ export default class Map extends Component {
           "icon-allow-overlap": true
         }
       });
+      if(this.props.mapbox3d) {
+        this.map.addSource('mapbox-dem', {
+          'type': 'raster-dem',
+          'url': 'mapbox://mapbox.mapbox-terrain-dem-v1',
```

Comment:
I saw the note that this only works for online maps. Should we be concerned about this source / layer attempting to be added when offline?